src/services/scheduler-api.ts

```diff
@@ -1,46 +1,19 @@
-import childProcess from 'child_process'
-import { readdirSync, } from 'fs'
-
-import { v4 as uuidv4, } from 'uuid'
-
-import * as scheduler from './scheduler'
-import logger from './logger'
-
-// We have to test the current directory if it's Typescript or Javascript. When running in `ts-node`
-// mode I do not expect childProcess to actually work, but also in that situation we don't need to
-// put the workers into a separate process for Kubernetes' sake since that's only during testing,
-// so in that case, we can just pass-through re-export the graphile worker scheduler.
-const files = readdirSync(__dirname);
-const isTs = files.some(file => /.*ts$/.test(file));
-const isChild = process.connected;
-const shouldRpc = !isTs && !isChild;
-
-const child = shouldRpc ? childProcess.fork(`${__dirname}/scheduler.js`) : undefined;
-
-// Primitive RPC handler. Only the parent process can trigger an RPC call, so we can track it all
-// here. Currently-active requests are turned into promises that are eventually resolved by the
-// child process' return message, identified by a unique ID each message includes at the end
-const messagePromises: { [key: string]: any[], } = {};
-
-function simpleRpc(fn: string, ...args: string[]) {
-  const id = uuidv4();
-  const p = new Promise((resolve, reject) => {
-    messagePromises[id] = [resolve, reject];
-  });
-  child?.send?.([fn, ...args, id]);
-  return p;
+import fetch from 'node-fetch';
+import logger from '../services/logger';
+
+const schedulerAddress = 'http://localhost:14527';
+
+async function fetchOrRaise(url: string) {
+  const response = await fetch(url);
+  if (response.ok) {
+    return
```

Comment:
Should this be `return response;` instead?

Reply:
On success, there's no response, just a `OK` text with 200 status code. Do we need the response object anyways?

Reply:
At the moment, no, so we can refactor later. It may just be surprising to write a new scheduler function that returns values back to the caller and they're inaccessible.

Reply:
Good point. Changing it as you proposed.